### DISCUSSION TITLE
Welcome to BibiGPT Discussions!

### DISCUSSION THREAD

**Discussion by JimmyLv** [JimmyLv/BibiGPT#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hfnuser0000** [comment]
Firstly, I wanted to say that I'm very impressed with your GitHub project that takes videos and outputs summaries. It's a really innovative and useful tool. I'm curious to learn more about the technology behind it, particularly how it's able to generate video summaries despite GPT-3.5 Turbo only being able to take text as its input. Your project seems to be pushing the boundaries of what is possible with AI, and I would love to hear more about the methodology or techniques used in its development. Could you please share some insights into how your project works, and how it differs from traditional text-based input? Thank you for your time and for creating such an exciting project!
